ragna/source_storages/_chroma.py

```diff
@@ -25,6 +28,9 @@ def __init__(self) -> None:
 
         import chromadb
 
+        self._embedding_model = MiniLML6v2()
+        self._chunking_model = NLTKChunkingModel()
```

Comment:
Why would the source storage need any of these?

Reply:
It doesn't, and these have been removed.